.github/workflows/pr-bloaty.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Compile Bloaty
         run: |
           git clone https://github.com/google/bloaty.git
-          cmake -B bloaty/build -G Ninja -S bloaty
+          cmake -B bloaty/build -S bloaty
```

Comment:
This will AFAIK build serially which we probably do not want.

Reply:
Fixed.